### PR TITLE
Verification checks against VM (Tasks for the user)

### PR DESCRIPTION
**Pull request by maxsva**
**Task193 / The ability to verifiy different things on VMs*:
add tasks(commands) to scenario

**Task193 / Run multiple commands on VMs**:
run VM in go routine, run commands in limited count(with Semaphore) of go routine 

fixes https://github.com/hobbyfarm/hobbyfarm/issues/193

### CODE REVIEW

File: pkg/controllers/vmsetcontroller/vmsetcontroller.go [hunk 1]
```diff
@@ -345,22 +344,28 @@ func (v *VirtualMachineSetController) reconcileVirtualMachineSet(vmset *hfv1.Vir
...
 			err = util.VerifyVM(v.vmLister, vm)
 			if err != nil {
 				glog.Error(err)
 			}
 		}
-
-		//TODO handle case of scaling down VMSets
 	}
-
+//-----------------------handle case of scaling down VMSets
```
Comment (jggoebel) [inline pkg/controllers/vmsetcontroller/vmsetcontroller.go:357]: this is of you old PR. Next time start of with the main branch of the project :P

File: pkg/scenarioserver/scenarioserver.go [hunk 2]
```diff
@@ -697,6 +698,18 @@ func (s ScenarioServer) CreateFunc(w http.ResponseWriter, r *http.Request) {
 	scenario.Spec.Tags = tags
 	scenario.Spec.KeepAliveDuration = keepaliveDuration
 
+	rawVMTasks := r.PostFormValue("vm_tasks")
+	if rawVMTasks != "" {
```
Comment (jggoebel) [inline pkg/scenarioserver/scenarioserver.go:702]: in the Update Metod you validate the Input, here you do not validate the input. Better to build a function to valdiate VirtualMachineTasks and use it in both methods

**Comment by ebauman** [comment]
Please hold off on merging this until we sort out the machine prov stuff

**Comment by jggoebel** [comment]
@ebauman This PR changes nothing for VM provisioning. It only touches scenarios + shell server. But we could still hold this

**Comment by ebauman** [comment]
it's mostly a naming thing, i'd like to move away from "virtualmachine" and towards "machine". if we're going to issue a scenario v2 it should have those changes.

**Comment by jggoebel** [comment]
We might need to update how a task is viewed as successful.
Sometimes it is defined by the returned status code, sometimes by the output.

We should make it optional to use both methods, one of the methods alone should be also ok.

**Comment by jggoebel** [comment]
Closed in favor of https://github.com/hobbyfarm/gargantua/pull/180

**Pull request closed without merging**